src/client/ClientSnapRealm.h

```diff
@@ -25,6 +25,7 @@ struct SnapRealm {
   std::set<SnapRealm*> pchildren;
   utime_t last_modified;
   uint64_t change_attr;
+  bool is_snapdir_visible;
```

Comment:
```suggestion
  bool is_snapdir_visible = true;
```

Reply:
Also, print this in `ostream& operator<<(ostream& out, const SnapRealm& realm)`.

Reply:
its added already `out << " snapdir_visibility=" << realm.srnode.is_snapdir_visible();`

Reply:
> changed it to `" is_snapdir_visible " << realm.srnode.is_snapdir_visible()` in this [push](https://github.com/ceph/ceph/compare/f308e52d438481021c205002bfae1ad29ff5556a..8d1a1c9b3e6bd31b2fc1facc00d1072d76311aed)